components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -669,6 +681,102 @@ public boolean isRefreshTokenRenewalEnabled() {
         return supportedResponseTypes;
     }
 
+    private void parseRequestObjectConfig(OMElement requestObjectBuildersElem) {
+        if (requestObjectBuildersElem != null) {
+            Iterator<OMElement> iterator = requestObjectBuildersElem
+                    .getChildrenWithName(getQNameWithIdentityNS(ConfigElements.REQUEST_OBJECT_BUILDER));
+            while (iterator.hasNext()) {
+                OMElement requestObjectBuildersElement = iterator.next();
+                OMElement builderNameElement = requestObjectBuildersElement
+                        .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.BUILDER_NAME));
+                String builderName = null;
+                if (builderNameElement != null) {
+                    builderName = builderNameElement.getText();
+                }
+
+                OMElement requestObjectImplClassElement = requestObjectBuildersElement
+                        .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.REQUEST_OBJECT_IMPL_CLASS));
+                String requestObjectImplClass = null;
+                if (requestObjectImplClassElement != null) {
+                    requestObjectImplClass = requestObjectImplClassElement.getText();
+                }
+                requestObjectBuilderClassNames.put(builderName, requestObjectImplClass);
+            }
+        } else {
+            // if this element is not present, assume the default case.
+            log.warn("\'RequestObjectBuilders\' element not configured in identity.xml. " +
```

Comment:
Should this be a warning ? Are we packing the identity.xml with the default value ? Or else there isn't any value ?

Reply:
lets make this an info log and add default values to the identity.xml